ytmusicapi/mixins/explore.py

```diff
@@ -209,9 +209,13 @@ def get_charts(self, country: str = 'ZZ') -> Dict:
             ]))
         charts_categories = ['videos', 'artists']
 
-        has_songs = bool(self.auth)
         has_genres = country == 'US'
         has_trending = country != 'ZZ'
+        # songs section appears to no longer exist, extra length check avoids
```

Comment:
I suppose the previous check was too simple since you'd actually need to check for a premium subscriber instead of a just a logged-in user. Should probably add a separate account to the tests that is authenticated but not subscribed

So I think we can just remove the `bool(self.auth)` part altogether since it does not actually make a difference

Reply:
I was indeed performing all tests using non-premium accounts.

Reply:
So we can remove `bool(self.auth) and` ?